aada/login.py

```diff
@@ -55,7 +55,7 @@ def __init__(self, session, role=None, account=None, saml_request=None):
         self._azure_mfa = self._config.get('azure_mfa')
         self._azure_kmsi = self._config.get('azure_kmsi', False)
         self._azure_username = self._config.get('azure_username')
-        self.browser = launch()
+        self.browser = launch(executablePath='/usr/bin/chromium-browser', args= ['--no-sandbox', '--headless', '--disable-gpu'])
```

Comment:
args are ok, but executablePath?? It is different based on OS, I used that also on windows and there is no /usr/bin/chromium... Is that needed here?